cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -322,4 +322,15 @@ class ReadFileTest {
 
         assertValues(expected, actual)
     }
+
+    @Test
+    fun readMarkdownTable() {
+        writeFile("table.md", "| Name | Age | Tenure |\n" + "| ---- | --- | ------ | \n" + "| \"a\"  | 24  | 1      | \n" + "| \"b\"  | 45  | 6      | \n" + "| \"a\"  | 24  | 2      |  ")
+
+        var args = listOf("\"${dirPath("table.md")}\"").map { it.exprValue() }
```

Comment:
Can we declare `args` using `val` instead of `var`? Generally, we prefer to use immutable variable.